doc/security/vulnerabilities.rst

```diff
@@ -11,15 +11,380 @@ sufficient time to fix them.  Because the release notes are locked to
+CVE-2020-10019
+--------------
+
+Buffer Overflow vulnerability in USB DFU of zephyr allows a USB
+connected host to cause possible remote code execution.
+
+This has been fixed in releases v1.14.2, v2.2.0, and v2.1.1.
+
+- `CVE-2020-10019 <http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-10019>`_
+
+- `Zephyr project bug tracker ZEPSEC-25
+  <https://zephyrprojectsec.atlassian.net/browse/ZEPSEC-25>`_
+
+- `PR23460 fix for 1.14.x
+  <https://github.com/zephyrproject-rtos/zephyr/pull/23460>`_
+
+- `PR23457 fix for 2.1.x
+  <https://github.com/zephyrproject-rtos/zephyr/pull/23457>`_
+
+- `PR23190 fix in 2.2.0
+  <https://github.com/zephyrproject-rtos/zephyr/pull/23190>`_
+
+CVE-2020-10021
+--------------
+
+Out-of-bounds write in USB Mass Storage with unaligned sizes
+
+Out-of-bounds Write in the USB Mass Storage memoryWrite handler with
+unaligned Sizes.
+
+See NCC-ZEP-024, NCC-ZEP-025, NCC-ZEP-026
+
+This has been fixed in releases v1.14.2, and v2.2.0.
+
+- `CVE-2020-10021 <https://docs.zephyrproject.org/latest/security/vulnerabilities.html#CVE-2020-10021>`_
+
+- `Zephyr project bug tracker ZEPSEC-26
+  <https://zephyrprojectsec.atlassian.net/browse/ZEPSEC-26>`_
+
+- `PR23455 fix for v1.14.2
+  <https://github.com/zephyrproject-rtos/zephyr/pull/23455>`_
+
+- `PR23456 fix for the v2.1 branch
+  <https://github.com/zephyrproject-rtos/zephyr/pull/23456>`_
+
+- `PR23240 fix for v2.2.0
+  <https://github.com/zephyrproject-rtos/zephyr/pull/23240>`_
+
+CVE-2020-10022
+--------------
+
+UpdateHub Module Copies a Variable-Size Hash String Into a Fixed-Size Array
+
+A malformed JSON payload that is received from an UpdateHub server may
+trigger memory corruption in the Zephyr OS. This could result in a
+denial of service in the best case, or code execution in the worst
+case.
+
+See NCC-ZEP-016
+
+This has been fixed in the below pull requests for master, branch from
+v2.1.0, and branch from v2.2.0.
+
+- `CVE-2020-10022 <https://docs.zephyrproject.org/latest/security/vulnerabilities.html#CVE-2020-10022>`_
+
+- `Zephyr project bug tracker ZEPSEC-28
+  <https://zephyrprojectsec.atlassian.net/browse/ZEPSEC-28>`_
+
+- `PR24154 fix for master
+  <https://github.com/zephyrproject-rtos/zephyr/pull/24154>`_
+
+- `PR24065 fix for branch from v2.1.0
+  <https://github.com/zephyrproject-rtos/zephyr/pull/24065>`_
+
+- `PR24066 fix for branch from v2.2.0
+  <https://github.com/zephyrproject-rtos/zephyr/pull/24066>`_
+
+CVE-2020-10023
+--------------
+
+Shell Subsystem Contains a Buffer Overflow Vulnerability In
+shell_spaces_trim
+
+The shell subsystem contains a buffer overflow, whereby an adversary
+with physical access to the device is able to cause a memory
+corruption, resulting in denial of service or possibly code execution
+within the Zephyr kernel.
+
+See NCC-ZEP-019
+
+This has been fixed in releases v1.14.2, v2.2.0, and in a branch from
+v2.1.0,
+
+- `CVE-2020-10023 <https://docs.zephyrproject.org/latest/security/vulnerabilities.html#CVE-2020-10023>`_
```

Comment:
ditto, just review these links across the document.

Reply:
Yeah, this was wrong in the script. I'll fix these by hand.